pubsub/rabbitmq/metadata.go

```diff
@@ -211,6 +222,12 @@ func (m *metadata) formatQueueDeclareArgs(origin amqp.Table) amqp.Table {
 		origin[argMaxLengthBytes] = m.maxLenBytes
 	}
 
+	if m.defaultQueueTTL != nil {
+		// Value in ms
+		ttl := *m.defaultQueueTTL / time.Millisecond
```

Comment:
I don't understand why this is dividing by MS?

if `defaultQueueTTL` is in seconds, to get the number of milliseconds, shouldn't you do `ttl := *m.defaultQueueTTL * time.Millisecond`? Unless I misunderstand

Reply:
You can literally call `*m.defaultQueueTTL.Milliseconds()`

Reply:
`m.defaultQueueTTL` is in time.Time format which is formatted when parsing.

> You can literally call *m.defaultQueueTTL.Milliseconds()

I will change to this, for it is easier to understand.

Reply:
done